caom2repo/caom2repo/core.py

```diff
@@ -89,13 +89,17 @@
 from six.moves.urllib.parse import urlparse
 
 # from . import version as caom2repo_version
-from . import version
+from caom2repo import version
```

Comment:
I believe these are equivalent. I've only made the change because if I want to debug the application with the debugger I need to turn this module into a script and by doing that the relative import stops working. AFAIK, relative imports are useful when rearranging the structure of the package around and I didn't find that appealing in this case.